site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -1,7 +1,5 @@
 ---
 api: tabs
-extra_permissions_html:
```

Comment:
`extra_permissions_html` only works for APIs that have a permissions derived from Chromium. The purpose of extra permissions, was to include additional permissions available to the API, like the **DeclarativeNetRequest API**.

I created `special_permissions_html` for the **Tabs API** and **Windows API**, because in these cases, there's no permissions derived from Chromium. Additionally, the extension can request either the `tabs` permission or host permissions to access the tabs privileged data (url, title, etc). This requires additional explanation that I will add in a separate PR.

Reply:
Just wanted to quickly note that I appreciate that you put in the extra effort to add this note to help clarify why it's being removed.